### PR TITLE
Update UserGuide.md

### PR DESCRIPTION
**Pull request by Benjykoh**
Add descriptions for commands "manual", "mark" and "unmark".

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -118,6 +120,34 @@ Example:
 * `task A0123456Z`
     * Lists out the tasks that student (A0123456Z) has.
     
+### Marking an undone task as done for a particular student: `mark`
+
+Marks a specific undone task as done for a particular student.
+
+Format : `mark n/STUDENT_NAME UNDONE_TASK_INDEX` (or) `mark i/STUDENT_ID UNDONE_TASK_INDEX`
```
Comment (Korochii) [inline docs/UserGuide.md:127]: I think we should only use student id since student name might not be unique?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -118,6 +120,34 @@ Example:
...
+* `mark n/John 1`
+    * Marks the first undone task for John as done.
+* `mark i/AXXXXXXXR 1`
+    * Marks the first undone task for the student with student ID AXXXXXXXR as done.
+
+### Marking a done task as undone for a particular student: `unmark`
+
+Marks a specific done task as undone for a particular student.
+
+Format : `unmark n/STUDENT_NAME DONE_TASK_INDEX` (or) `unmark i/STUDENT_ID DONE_TASK_INDEX`
```
Comment (Korochii) [inline docs/UserGuide.md:141]: Same as above.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -118,6 +120,34 @@ Example:
...
+### Marking an undone task as done for a particular student: `mark`
+
+Marks a specific undone task as done for a particular student.
+
+Format : `mark n/STUDENT_NAME UNDONE_TASK_INDEX` (or) `mark i/STUDENT_ID UNDONE_TASK_INDEX`
+
+* The undone task corresponding to the index or the particular student will be marked as done in the TAPA.
+  
+Example:
+* `mark n/John 1`
```
Comment (Korochii) [inline docs/UserGuide.md:132]: Same as above.

File: docs/UserGuide.md [hunk 4]
```diff
@@ -118,6 +120,34 @@ Example:
...
+### Marking a done task as undone for a particular student: `unmark`
+
+Marks a specific done task as undone for a particular student.
+
+Format : `unmark n/STUDENT_NAME DONE_TASK_INDEX` (or) `unmark i/STUDENT_ID DONE_TASK_INDEX`
+
+* The done task corresponding to the index for the particular student will be marked as undone in the TAPA.
+
+Example:
+* `unmark n/John 1`
```
Comment (Korochii) [inline docs/UserGuide.md:146]: Same as above.

File: docs/UserGuide.md [hunk 5]
```diff
@@ -127,6 +157,18 @@ Format: `archive`
 * A copy of the details currently saved in the address book will be saved to a separate file.
 * The file name will be the date and time of the archive operation.
 
+### Displaying manual for a command: `manual`
+
+Display the format for a specified command and a short description for that command.
+
+Format: `manual COMMAND_NAME`
+
+* The format for the command corresponding to the command name will be displayed, along with a short description
```
Comment (Korochii) [inline docs/UserGuide.md:166]:
```suggestion
* The format for the command corresponding to the command name will be displayed, along with a short description.
```

File: docs/UserGuide.md [hunk 6]
```diff
@@ -127,6 +157,18 @@ Format: `archive`
...
+
+Example
+* `manual add`
+    * Display the format for the command add, and briefly describes the command
```
Comment (Korochii) [inline docs/UserGuide.md:170]:
```suggestion
    * Display the format for the command add, and briefly describes the command.
```

File: docs/UserGuide.md (hunk 5 above)
Comment (Korochii) [reply docs/UserGuide.md:166]: Might want to include what the user should expect if an invalid command is supplied to `manual` (maybe an error message or smt?)

File: docs/UserGuide.md [hunk 7]
```diff
@@ -150,8 +192,11 @@ Action      | Format, Examples
 **Add**     | `add i/MATRICULATION_NO n/STUDENT_NAME m/MODULE_CODE [p/PHONE_NUMBER] [t/TELEGRAM_HANDLE] [e/EMAIL_ADDRESS] ` <br> e.g., `add i/A0123456Z n/john m/CS2103T p/98765432 t/@john e/E0123456Z`
 **Delete**    | `delete STUDENT_INDEX` <br> e.g., `delete 10`
 **Find**    | `find n/STUDENT_NAME` (or) `find i/STUDENT_ID` <br> e.g., `find n/john`, `find i/A0123456Z`
+**Manual**  | `manual COMMAND_NAME` <br> e.g., `manual add`
 **Exit**    | `exit`
 **Task**    | `task i/STUDENT_ID` <br> e.g., `task A0123456Z`
+**Mark**    | `mark n/STUDENT_NAME UNDONE_TASK_INDEX` (or) `mark i/STUDENT_ID UNDONE_TASK_INDEX` <br> e.g., `mark n/John 1`, `mark i/AXXXXXXXR 1`
+**Unmark**  | `unmark n/STUDENT_NAME DONE_TASK_INDEX` (or) `unmark i/STUDENT_ID DONE_TASK_INDEX` <br> e.g., `unmark n/John 1`, `unmark i/AXXXXXXXR 1`
```
Comment (Korochii) [inline docs/UserGuide.md:199]: Same as above.

**Review by Korochii — COMMENTED** [review]
Overall good job in including the necessary features in our user guide. However, there are some minor improvements that I have suggested in my review.

File: docs/UserGuide.md [hunk 8]
```diff
@@ -118,6 +120,34 @@ Example:
 * `task A0123456Z`
     * Lists out the tasks that student (A0123456Z) has.
     
+### Marking an undone task as done for a particular student: `mark`
+
+Marks a specific undone task as done for a particular student.
+
+Format : `mark n/STUDENT_NAME UNDONE_TASK_INDEX` (or) `mark i/STUDENT_ID UNDONE_TASK_INDEX`
+
+* The undone task corresponding to the index or the particular student will be marked as done in the TAPA.
```
Comment (Korochii) [inline docs/UserGuide.md:129]: Might want to include what the user should expect if an invalid index is supplied (maybe an error message or smt?)

File: docs/UserGuide.md [hunk 9]
```diff
@@ -118,6 +120,34 @@ Example:
...
+* `mark i/AXXXXXXXR 1`
+    * Marks the first undone task for the student with student ID AXXXXXXXR as done.
+
+### Marking a done task as undone for a particular student: `unmark`
+
+Marks a specific done task as undone for a particular student.
+
+Format : `unmark n/STUDENT_NAME DONE_TASK_INDEX` (or) `unmark i/STUDENT_ID DONE_TASK_INDEX`
+
+* The done task corresponding to the index for the particular student will be marked as undone in the TAPA.
```
Comment (Korochii) [inline docs/UserGuide.md:143]: Might want to include what the user should expect if an invalid index is supplied (maybe an error message or smt?)

File: docs/UserGuide.md (hunk 8 above)
Comment (Benjykoh) [reply docs/UserGuide.md:129]: Added what user should expect if an invalid index is supplied (when out of bounds and when task is already marked

File: docs/UserGuide.md (hunk 7 above)
Comment (Benjykoh) [reply docs/UserGuide.md]: Updated to only handle student ID.

File: docs/UserGuide.md (hunk 4 above)
Comment (Benjykoh) [reply docs/UserGuide.md]: Updated to only handle student ID.

File: docs/UserGuide.md (hunk 3 above)
Comment (Benjykoh) [reply docs/UserGuide.md]: Updated to only handle student ID.

File: docs/UserGuide.md (hunk 2 above)
Comment (Benjykoh) [reply docs/UserGuide.md]: Updated to only handle student ID.

File: docs/UserGuide.md (hunk 1 above)
Comment (Benjykoh) [reply docs/UserGuide.md]: Updated to only handle student ID.

**Review by Korochii — APPROVED** [review]
LGTM! 😄

**Pull request merged**